fabric-object-builder-api-v1/src/testmod/java/net/fabricmc/fabric/test/object/builder/VillagerTypeTest1.java

```diff
@@ -49,12 +49,6 @@ public void onInitialize() {
 		});
 
 		CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> {
-			dispatcher.register(literal("fabric_refreshtrades").executes(context -> {
-				TradeOfferHelper.refreshOffers();
-				context.getSource().sendFeedback(Text.literal("Refreshed trades"), false);
-				return 1;
-			}));
```

Comment:
Hmm, why was this removed

Reply:
The method does nothing. (Should this be `forRemoval=true`'d)?

Reply:
Yeah it was deprecated in 1.16 before `forRemoval=true` was a thing, see #1430. This is one of the instances where the method is so broken we should deprecate it for removal.